### ISSUE TITLE
Local development

### ISSUE THREAD

**Issue by Gazook89** [tensiontides/Fight2HB#1]
I gave this a spin and noticed some places where there could be improvement in the output, but hate to be the guy who nitpicks without offering to do some of the work.  However, I'm not very familiar with python, and couldn't figure out how to run this just locally on my machine (macOS, using VS Code).  Any quick tips on doing that?  

Otherwise, the big suggestion I have is to update the output to reflect Homebrewery's new v3 Renderer requirements for a stat block.  Here is an example of that new output:

```
{{monster,frame
## Angel Pigeon
*Medium annoyance, chaotic sloppy*
___
**Armor Class** :: 11 (chain mail, shield)
**Hit Points**  :: 57(1d4 + 5)
**Speed**       :: 29ft.
___
|  STR  |  DEX  |  CON  |  INT  |  WIS  |  CHA  |
|:-----:|:-----:|:-----:|:-----:|:-----:|:-----:|
|7 (-1)|7 (-1)|3 (-3)|6 (-2)|8 (-1)|7 (-1)|
___
**Condition Immunities** :: None
**Senses**               :: darkvision 60 ft., passive Perception 14
**Languages**            :: Latin
**Challenge**            :: 8 (8447 XP)
___
***Enormous Nose.*** This creature gains advantage on any check involving putting things in its nose.
:
***Fowl Appearance.*** While the creature remains motionless, it is indistinguishable from a normal chicken.
:
***Enormous Nose.*** This creature gains advantage on any check involving putting things in its nose.
:
***Pack Tactics.*** These guys work together like peanut butter and jelly.
:
***Hangriness.*** This creature is angry, and hungry. It will refuse to do anything with you until its hunger is satisfied.

When in visual contact with this creature, you must purchase an extra order of fries, even if they say they aren't hungry.
### Actions
***Team Elbow.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
:
***Gory Body Sweep.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
:
***Turnbuckle Roll.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
}}
```

Basically it removes the need for an `<hr>` followed by a `<blockquote>` and instead just uses a new custom markdown token `{{ ... }}` to create `<div></div>`, with a few other changes.

I can likely provide this new parsing but just need to get a local app up and running.
